plugins/aggregators/histogram/README.md

```diff
@@ -39,16 +40,20 @@ of the algorithm which is implemented in the Prometheus
   ## of accumulating the results.
   reset = false
 
+  ## Whether bucket values should be accumulated. If set to false, "gt" tag will be added.
+  ## Defaults to true.
+  cumulative = true
```

Comment:
What's going to be the value if this isn't set in a user's config?

Reply:
If `cumulative` isn't set in config, it defaults to true:

https://github.com/influxdata/telegraf/blob/55ad4fb3ebdb1e106f7c46ee3ff131920086fbfb/plugins/aggregators/histogram/histogram.go#L67-L75